src/lib/infrastructure/Services/email_service.ts

```diff
@@ -0,0 +1,53 @@
+import * as i from '../interfaces'
+import {injectable, inject} from 'inversify'
+import nodemailer from 'nodemailer'
+
+
+
+
+@injectable()
+export default class EmailService implements i.IEmailService {
+    
+    constructor( )
+    {
+
+    }
+
+    sendVerificationEmail(recepient:String):Promise<Object>{
+
+    const nodemailer = require('nodemailer');
+
+
+    var transporter = nodemailer.createTransport({
+        service: 'gmail',
+        auth: {
+            user: 'internviewpr@gmail.com', 
```

Comment:
Move to .env.local and let us know in clickup or whatsapp of email credentials

Reply:
Estoy verificando porque cuando uso los credenciales guardados en mis env variables me sale este problema de autenticacion: 
    Error: connect ECONNREFUSED 127.0.0.1:587
        at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {
      errno: 'ECONNREFUSED',
      code: 'ESOCKET',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 587,
      command: 'CONN'
    }